recipes/pygenometracks/meta.yaml

```diff
@@ -25,10 +25,12 @@ requirements:
     - intervaltree >=2.1.0
     - pybigwig >=0.3.16
     - future >=0.17.0
-    - hicmatrix >=12
+    - hicmatrix >=13
     - pysam >=0.14
     - gffutils >=0.9
     - tqdm >=4.20
+    - libopenblas <0.3.10
```

Comment:
Is this needed because of our old matplotlib version? We should not do this. This will prevent from creating mixed environments with other software.

Reply:
@lldelisle says there is a bug in version 0.3.10, therefore we have to fix the version until there is a bug free update.

Reply:
And matplotlib version stays as it is tested, we have made too many bad experiences with (minor) updates from that library.

Reply:
Where is libopenblas coming from, which package does need it? What is the error? Can you add a test here that exercises it?

Reply:
I don't know where it comes from but it raises segmentation faults if I don't put this constrains:
https://travis-ci.com/github/lldelisle/pyGenomeTracks/builds/176051693